modin/pandas/test/test_dataframe.py

```diff
@@ -2549,10 +2549,105 @@ def test_resample(self, rule, axis, closed, label, on, level):
                 pandas_resampler.aggregate(["sum", "mean", "max"]),
             )
 
-    def test_rolling(self):
-        df = pd.DataFrame({"B": [0, 1, 2, np.nan, 4]})
-        with pytest.warns(UserWarning):
-            df.rolling(2, win_type="triang")
+    @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+    @pytest.mark.parametrize("window", [5, 100])
+    @pytest.mark.parametrize("min_periods", [None, 5])
+    @pytest.mark.parametrize("win_type", ["triang", None])
+    def test_rolling(self, data, window, min_periods, win_type):
```

Comment:
Do you think it would be worth adding a new file called `test_rolling.py`? That way we can keep all of the tests for rolling separate instead of testing all functionality in one test, but it won't make `test_dataframe.py` longer. 

When you add it, make sure you also add to CI.

Reply:
Ok, I'll do this in next commit.